<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenCap</class>
 <widget class="QWidget" name="ScreenCap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>190</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>190</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_panel_name">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>21</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgba(243, 249, 255, 10);
}</string>
   </property>
   <property name="text">
    <string>   Screen capture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_screenshot">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Take screenshot</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 1px solid rgba(255, 255, 255, 0);
	background-color: rgba(255, 255, 255, 0);
	color: rgb(255, 0, 19);
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #404040 stop: 1 #686868);
}</string>
   </property>
   <property name="text">
    <string>üì∏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_record">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Record GIF</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 1px solid rgba(255, 255, 255, 0);
	background-color: rgba(255, 255, 255, 0);
	color: rgba(235, 65, 50, 1.0);
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #404040 stop: 1 #686868);
}</string>
   </property>
   <property name="text">
    <string>‚¨§</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_open_folder">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>34</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Open media folder</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 1px solid rgba(255, 255, 255, 0);
	background-color: rgba(255, 255, 255, 0);
	color: rgb(255, 0, 19);
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #404040 stop: 1 #686868);
}
</string>
   </property>
   <property name="text">
    <string>üìÅ</string>
   </property>
  </widget>
  <widget class="Line" name="separator_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_folder_path">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>161</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>161</width>
     <height>21</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Choose media path</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 2px gray ;
	background-color: rgba(49, 169, 82, 0.5);
	border-radius: 10px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #404040 stop: 1 #686868);
}</string>
   </property>
   <property name="text">
    <string>Path</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_folder_path">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>147</red>
         <green>147</green>
         <blue>149</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>208</red>
         <green>208</green>
         <blue>208</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="96">
         <red>164</red>
         <green>166</green>
         <blue>168</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Folder to save media..</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_screenshot_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Choose screenshot format</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	background-color: rgba(0, 0, 0, 0.0);
	color: rgba(255, 255, 255, 0.7);
	border: 1px solid rgba(255, 255, 255, 0);
	selection-background-color: rgba(255, 255, 255, 30);
}</string>
   </property>
   <item>
    <property name="text">
     <string>JPEG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BMP</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_screenshot_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>147</red>
         <green>147</green>
         <blue>149</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>208</red>
         <green>208</green>
         <blue>208</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="96">
         <red>164</red>
         <green>166</green>
         <blue>168</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Screenshot format</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
